components/ProjectCard.js

```diff
@@ -1,13 +1,16 @@
-import titles from "lib/db.json";
-import { v4 as uuidv4 } from "uuid";
+import { projects } from "@/lib/db.js";
 import styled from "styled-components";
+import Link from "next/link";
+import { useRouter } from "next/router";
 
 export default function ProjectCard() {
   return (
     <>
```

Comment:
You don't need these fragments. Please use them only if you want to return two or more elements "side by side" and not wrapped in a container element.